mackerel-plugin-mssql/lib/mssql_other.go

```diff
@@ -0,0 +1,8 @@
+// +build !windows
+
+package mpmssql
+
+// Do the plugin
+func Do() {
+	panic("The mackerel-plugin-mssql does not work on non Windows environment, of course.")
```

Comment:
I hope this plugin will also work in other platforms IN FUTURE, since SQL Server >= 2017 works in Linux :-)